crates/ruma-state-res/src/lib.rs

```diff
@@ -238,28 +238,28 @@ fn reverse_topological_power_sort<E: Event>(
 
     lexicographical_topological_sort(&graph, |event_id| {
         let ev = fetch_event(event_id).ok_or_else(|| Error::NotFound("".into()))?;
-        let pl = event_to_pl.get(event_id).ok_or_else(|| Error::NotFound("".into()))?;
-
-        debug!("{:?}", (-*pl, ev.origin_server_ts(), &ev.event_id()));
-
-        // This return value is the key used for sorting events,
-        // events are then sorted by power level, time,
-        // and lexically by event_id.
-        Ok((-*pl, ev.origin_server_ts(), ev.event_id().to_owned()))
+        let pl = *event_to_pl.get(event_id).ok_or_else(|| Error::NotFound("".into()))?;
+        Ok((pl, ev.origin_server_ts()))
     })
 }
 
 /// Sorts the event graph based on number of outgoing/incoming edges.
 ///
-/// `key_fn` is used as a tie breaker. The tie breaker happens based on power level, age, and
-/// event_id.
+/// `key_fn` is used as to obtain the power level and age of an event for breaking ties.
```

Comment:
```suggestion
/// `key_fn` is used as to obtain the power level and age of an event for breaking ties. (Together with the event ID)
```

Reply:
Also, hold on, if i'm reading the rest of the changes correctly, the return value for this has been flipped from negative back to positive?

This is a bit of a subtle change, and you should document that.

Reply:
Yeah, return value is flipped but all existing consumers of this API actually return 0 right now, so I don't think writing docs will help anybody.

Reply:
Conduit doesnt use positive or negative for this (in the only instance, it provided a 0), but this is still worth noting.

Reply:
Added a changelog entry.

Reply:
Just thought it might be worth noting that when conduit calls `resolve` it does use the negative power level so this has been working